contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -390,69 +451,184 @@ contract AerodromeAMOStrategy is InitializableAbstractStrategy {
-            (uint256 mintedTokenId,,uint256 amountWETH, uint256 amountOETHb) = positionManager.mint(
+            (uint256 mintedTokenId, uint128 liquidity, uint256 WETHAmountSupplied, uint256 OETHbAmountSupplied) = positionManager.mint(
                 INonfungiblePositionManager.MintParams({
                     token0: WETH,
                     token1: OETHb,
                     tickSpacing: 1,
-                    tickLower: 0,
-                    tickUpper: 1,
-                    amount0Desired: wethBalance,
-                    amount1Desired: 0,
+                    tickLower: lowerTick,
+                    tickUpper: upperTick,
+                    amount0Desired: WETHBalance,
+                    amount1Desired: OETHbRequired,
+                    // we don't need slippage protection checking for that in _checkLiquidityWithinExpectedShare
                     amount0Min: 0,
+                    // we don't need slippage protection checking for that in _checkLiquidityWithinExpectedShare
                     amount1Min: 0,
                     recipient: address(this),
                     deadline: block.timestamp,
                     // needs to be 0 because the pool is already created
                     // non zero amount attempts to create a new instance of the pool
-                    sqrtPriceX96: 0 
+                    sqrtPriceX96: 0
                 })
             );
+
             tokenId = mintedTokenId;
-            // TODO add incerase liquidity event
-            netValue += amountWETH + amountOETHb;
+
+            updateNetValue();
+            emit LiquidityAdded(
+                WETHBalance, // WETHAmountDesired
+                OETHbRequired, // OETHbAmountDesired
+                WETHAmountSupplied, // WETHAmountSupplied
+                OETHbAmountSupplied, // OETHbAmountSupplied
+                mintedTokenId, // tokenId
+                netValue
+            );
         } else {
 
-            (,uint256 amountWETH, uint256 amountOETHb) = positionManager.increaseLiquidity(
+            (,uint256 WETHAmountSupplied, uint256 OETHbAmountSupplied) = positionManager.increaseLiquidity(
                 INonfungiblePositionManager.IncreaseLiquidityParams({
                     tokenId: tokenId,
-                    amount0Desired: wethBalance,
-                    amount1Desired: 0,
+                    amount0Desired: WETHBalance,
+                    amount1Desired: OETHbRequired,
+                    // we don't need slippage protection checking for that in _checkLiquidityWithinExpectedShare
                     amount0Min: 0,
+                    // we don't need slippage protection checking for that in _checkLiquidityWithinExpectedShare
                     amount1Min: 0,
                     deadline: block.timestamp
                 })
             );
-            // TODO add incerase liquidity event
-            netValue += amountWETH + amountOETHb;
+
+            updateNetValue();
```

Comment:
⚪️ The phrase `netValue` implies that it's built out of things adding to it and things subtracting from it. This used to be perfectly accurate, but is not longer the case. We should probably change the variable name.

Reply:
Thanks good point. will rename to underlyingAssets